### DISCUSSION TITLE
Custom account verification page

### DISCUSSION THREAD

**Discussion by laszlo1337** [pocketbase/pocketbase#3934] [Q&A] [answered]
Hello. 

Is it possible to replace or at least style account verification page in standalone pocketbase?

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, you can change the email confirm verification url to a custom one from the _Admin UI > Settings > Mail settings > {:template} > "Action URL" input_ (the only required parameter is `{TOKEN}`). 

When the user click on the link in the verification email they will be redirected to your page, where you'll have to extract the token from the url (you can pass it either as path or query param) and then send a [Confirm verification](https://pocketbase.io/docs/api-records/#confirm-verification) request with it.

**Reply by laszlo1337** [reply]
Thank you very much! And also for such fast response!

**Reply by laszlo1337** [reply]
I have one more question, in case I'd want to send emails in two languages, what should I do?

**Reply by ganigeorgiev** [reply]
There is no builtin solution for this at the moment (see https://github.com/pocketbase/pocketbase/issues/593).

The recommendations for now are:
- have both languages in the same template one under the other (eg. separated with `---` or `<hr>`)

- or overwrite the default system emails via hook and manually switch the text based on your user language (the last part would differ depending on how do you store this info)

**Reply by laszlo1337** [reply]
Since I don't want to expose PB to the outside world I plan on keeping it running on localhost on my server and allow access only to my app. This turns out to be problematic with email verification template, since it will use the localhost address for the link in email message. I cannot not use it, because validation says ACTION_URL is required. 

essentially I just wanted to provide link to my domain and just use {TOKEN} instead of full {ACTION_URL} 
like this `mydomain.com/verify-account?token={TOKEN}`

What else can I do? 

EDIT: I got it - looks like "application url" is what does it - but I don't know if it won't affect anything else since I thought this application url defines where the app will be run? probably not
